pkg/codegen/python/doc.go

```diff
@@ -37,9 +37,22 @@ func (d DocLanguageHelper) GetDocLinkForPulumiType(pkg *schema.Package, typeName
 	return ""
 }
 
-// GetDocLinkForResourceType is not implemented at this time for Python.
+// GetDocLinkForResourceType returns the Python API doc for a type belonging to a resource provider.
 func (d DocLanguageHelper) GetDocLinkForResourceType(pkg *schema.Package, modName, typeName string) string {
-	return ""
+	var path string
+	switch {
+	case pkg.Name != "" && modName != "":
+		path = fmt.Sprintf("pulumi_%s/%s", pkg.Name, modName)
+	case pkg.Name == "" && modName != "":
+		// The k8s module names contain the domain names. For now we are stripping them off manually so they link correctly.
+		modName = strings.TrimSuffix(modName, ".k8s.io")
```

Comment:
We should do this in the preceding `case` statement as well :) However, since the string can't be `nil` here, let's just do this before we enter the `switch-case` expression itself. You could just add a check for `modName != ""` and do the string replace there.

Reply:
Here's a bad [URL](http://content-bucket-e866e28.s3-website-us-east-1.amazonaws.com/docs/reference/pkg/python/pulumi_kubernetes/admissionregistration.k8s.io/v1/#MutatingWebhookConfiguration) I got from the resource page for [MutatingWebhookConfiguration](http://content-bucket-e866e28.s3-website-us-east-1.amazonaws.com/docs/reference/pkg/kubernetes/admissionregistration/v1/mutatingwebhookconfiguration/).

Reply:
yeah, good point.  lol. 🤦